usagov/usagov-2021
# USAGOV-1757-autologout-patch: Add missing autologout configs.

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1757

## Description

Add configs created with module upgrade to autologout v2.0.

## Type of Changes

  - Add screenshot showing what it should look like
- [ X ] Drupal Config (requires "drush cim")

## Testing Instructions


1. Navigate to /admin/config/people/autologout.
2. Change the "Timeout value in seconds" setting to 60 seconds.
3. Wait 60 seconds.
4. You are logged out.
5. Wait 20 seconds.
6. You are redirected to the sign in page.

Make sure to cleanup by navigating back to /admin/config/people/autologout and setting the "Timeout value in seconds" setting to 1800 seconds.

### Change Requirements



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
